src/encoded/schemas/human_donor.json

```diff
@@ -64,20 +74,44 @@
                     }
                 }
             ]
+        },
+        "twin_type": {
+            "comment": "Twin type could be specified only if twin sibling is specified.",
+            "required": ["twin"]
         }
-
     },
     "properties": {
         "schema_version": {
-            "default": "9"
+            "default": "10"
         },
         "ethnicity": {
             "title": "Ethnicity",
-            "type": "string"
+            "comment": "We use http://bioportal.bioontology.org/ontologies/SNOMED-Ethnic-Group as a template for the enum list of ethnicities.",
+            "type": "string",
+            "enum": [
+                "African American",
+                "Arab",
+                "Arab Indian",
+                "Asian",
+                "Asian Hawaiian Eskimo",
+                "Black",
```

Comment:
So SNOMED has both "Black" and "African-American"?

Reply:
yes they do (African American is a subclass of Black)